arangod/IResearch/IResearchRocksDBRecoveryHelper.cpp

```diff
@@ -112,13 +112,17 @@ bool IResearchRocksDBRecoveryHelper::skip(Impl& impl) {
 
 template<bool Force>
 void IResearchRocksDBRecoveryHelper::clear() noexcept {
-  if constexpr (!Force) {
+  if constexpr (Force) {
+    if (!_ranges.empty()) {
+      _ranges.clear();
+    }
+  } else {
     if (_indexes.size() < kMaxSize && _links.size() < kMaxSize &&
         _ranges.size() < kMaxSize) {
       return;
```

Comment:
I guess it is intentional that we don't clear `_ranges` here anymore, but please confirm.

Reply:
we clear

Reply:
![image](https://github.com/arangodb/arangodb/assets/32071355/31060b6e-95c3-4195-80c8-d9de84af2cbf)
line 124